src/overlords/defense/outpostDefense.ts

```diff
@@ -27,6 +27,9 @@ export class OutpostDefenseOverlord extends CombatOverlord {
 
 	private handleCombat(zerg: CombatZerg): void {
 		if (this.room && this.room.hostiles.length == 0) {
+			if (zerg.pos.getRangeTo(this.directive.pos) > 4) {
```

Comment:
forcing the zerg to regroup like this makes them vulnerable to edge dancing attacks -- at the very least there should be a timer before regrouping activates